### ISSUE TITLE
Questions on data format and data reading

### ISSUE THREAD

**Issue by dschwilk** [EmmaLiTsai/recoverKBTDR#18]
Names and format of data file columns seems inconsistent. I'm referring to the example trace files and the file reading code, `read_trace()`

Some questions:

1. Why does the trace file have 4 columns. I would expect 2.
2. Why does the time dots file have row numbers in the file? Sees unnecessary and is inconsistent with other csv files.
3. The files themselves use capital X and Y as column names. I'd probably have chosen lowercase but that is fine. But then the time_dots column gets renamed during `tidy_timedots()` as "x_val"? Why? This is confusing. Why not keep names consistent? IN fact, I don't like that tidy_timedots function. I think the function is unnecessary and that `read_trace` should handle this if it is really low level and done every time.  Why are you adding in a fake time dot at 0? Why not just handle that when you use the time dots in center_trace? The latter seems easier to maintain and avoids fake data. This caught me out because I knew what time dots should be but the data being handed to `center_trace()` was not what was in the time_dots csv file.

**Comment by dschwilk** [comment]
Also, this is an unnecessary use of dplyr:

```
  time_dots$Y <- dplyr::case_when(time_dots$Y < 0 ~ abs(time_dots$Y),
                                  time_dots$Y > 0 ~ -(time_dots$Y), 
                                  time_dots$Y == 0 ~ (time_dots$Y))
```
Why not just switch sign?
```
time_dots$Y <- -time_dots$Y
```
That is all you are doing.

I'm confused by this. Why is the time dots file coming in reversed relative to the trace? I think this function should be removed.

**Comment by EmmaLiTsai** [comment]
Thank you for looking over my code! I will fix the errors you pointed out with the file names and the unnecessary use of dplyr-- it is very helpful to have an extra set of eyes on this. 

To answer your questions: 

1) There were two extra columns that I added during image processing that are unnecessary and mainly for my own data exploration. I can remove these columns from the sample files. 

2) The extra column in the time dots data frame is from when I called the read.csv() function using lapply() without the row.names = 1 argument. I will edit the read_trace() function to account for this and read the time dots file in properly. 

3) The X and Y column names are defaults from ImageJ that I didn't like, and therefore I changed them in the trace and time dots tidying functions. I can add this change to the read_trace() function and make them consistent across files. The extra time dot was added because it was conceptually easier for me to create code to center the scan, and also so I could assign times to points at the very beginning of the record. The previous team from the 90's also started tracking time at the very beginning of  the record, so I attempted to model code after their previous methods. I can remove it and attempt to handle this within the centering and time assignment functions.
